jump-leveleditor/src/main/java/com/bitdecay/jump/leveleditor/utils/LevelUtilities.java

```diff
@@ -17,10 +17,13 @@ public static Level loadLevel(String fileName) {
 	}
 
 	public static Level loadLevel(File file) {
-
-		return null;
+		return FileUtils.loadFileAs(Level.class, file);
```

Comment:
Also, I wasn't sure if this was intentional or not, but I took out the return null and put in a proper call to FileUtils.  If it is intentional, then maybe we should just remove the method all together.